src/store/actions/contact.actions.ts

```diff
@@ -50,6 +54,20 @@ export const createContact = (args: ICreateContactArgs): ThunkAction<Promise<voi
   };
 };
 
+export const updateContact = (args: IUpdateContactArgs): ThunkAction<Promise<void>, RootState, unknown, Action> => {
+  return async (dispatch: ThunkDispatch<RootState, unknown, Action>) => {
+    dispatch({type: CONTACTS_LOADING});
+    editContact(args)
+      .then(() =>
```

Comment:
i would give this a result and then use that as the payload, now there could be a difference between the args and the actual updated contact

Reply:
channged